wordpress-mobile/WordPressAuthenticator-iOS
# [Login- "Continue with WordPress.com" flow] - Add What is WordPress.com? button.

Related to https://github.com/woocommerce/woocommerce-ios/issues/7212
Design ref: p1657014068195729/1656997606.750839-slack-C03L1NF1EA3

### Description
This PR adds `What is WordPress.com?` button in the Email entry screen of "Continue with WordPress.com" flow. 

As we cannot expect all users who install the app to be familiar with "WordPress.com" we are adding this button to teach users about "WordPress.com".  

This is part of "**Experiment 2: Add more information about the login process**" in pe5sF9-6C-p2

### Testing instructions 
Check out the branch from https://github.com/woocommerce/woocommerce-ios/pull/7213/ and follow the testing instructions specified in the PR. 

### Screenshots


| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/524475/177331100-1649a299-655b-47dc-ba14-f8ac5547dd0b.jpeg" width="400" /> | <img src="https://user-images.githubusercontent.com/524475/177331050-d53749ab-63dc-4009-8fe6-17541a8ff705.jpeg" width="400" />  |
